projects/sandbox/src/app/progress-tracker/progress-tracker.component.html

```diff
@@ -186,4 +195,100 @@ <h2 class="title-id">{{ datasetId }}</h2>
       </span>
     </ng-container>
   </div>
+  <ng-container *ngIf="progressData['tier-zero-info']; let tierInfo">
+    <div
+      class="warning-view"
+      *ngIf="
+        (tierInfo['content-tier'] && tierInfo['content-tier'].total) ||
+        (tierInfo['metadata-tier'] && tierInfo['metadata-tier'].total)
+      "
+      [ngClass]="{
+        closed: !warningViewOpen
+      }"
+    >
+      <sb-navigation-orbs
+        libClickAware
+        [ignoreClasses]="['link-internal', 'nav-orb', 'warning-view-content']"
+        [clickAwareIgnoreWhen]="!showing"
+        (clickOutside)="closeWarningView()"
+        ariaLabel="Tier Zero Warnings Navigation"
+        class="sb-navigation-orbs vertical zero-warning"
+        [count]="getOrbConfigCount()"
+        [fnClassMapInner]="getOrbConfigInner.bind(this)"
+        [fnClassMapOuter]="getOrbConfigOuter.bind(this)"
+        [tooltips]="[
+          'content-tier-zero records found (click to see samples)',
+          'metadata-tier-zero records found (click to see samples)'
+        ]"
+        (clickEvent)="setWarningView($event)"
+      ></sb-navigation-orbs>
+      <div
+        class="warning-view-content"
+        libClickAware
+        [ignoreClasses]="['link-internal', 'nav-orb', 'sb-navigation-orbs']"
+        [clickAwareIgnoreWhen]="!showing"
+        (clickOutside)="closeWarningView()"
+      >
+        <ng-template #warningViewHeader let-list="list" let-label="label">
+          <span class="warning-view-header">
+            This dataset contains
+            {{
+              list.length
```

Comment:
Shouldn't this be contentTier.total or metadataTier.total?
Otherwise this will not match the actual number of records with tier 0